src/main/java/org/spongepowered/common/item/inventory/SpongeItemStackSnapshot.java

```diff
@@ -141,7 +142,11 @@ public int getCount() {
 
     @Override
     public ItemStack createStack() {
-        return this.privateStack.copy();
+        net.minecraft.item.ItemStack nativeStack = ItemStackUtil.cloneDefensiveNative(ItemStackUtil.toNative(this.privateStack.copy()));
```

Comment:
I don't think all of this is necessary, as `cloneDefensiveNative` calls `ItemStack#copy`